sys/mips/include/cheri.h

```diff
@@ -171,6 +171,14 @@ cheri_get_cyclecount(void)
 	__asm__ __volatile__("li $0, 0xdead");				\
 } while(0)
 
+#define	CHERI_START_USER_TRACE	do {					\
```

Comment:
We should really rename this and move it out of cheri-specific code, but that can be another commit.